services/QuillLMS/app/assets/stylesheets/shared/table/react-table.scss

```diff
@@ -55,13 +124,10 @@
     }
 
     .rt-th {
-      background-color: #fafafa;
+      // background-color: #fafafa;
```

Comment:
Should this just be removed since it's commented out?

Reply:
yes, good catch